src/plugins/data/public/ui/search_bar/search_bar.tsx

```diff
@@ -332,7 +332,7 @@ class SearchBarUI extends Component<SearchBarProps, State> {
       if (this.props.onSaved) {
         this.props.onSaved(response);
       }
-    } catch (error) {
+    } catch (error: any) {
       this.services.notifications.toasts.addDanger(
         `An error occured while saving your query: ${error.message}`
```

Comment:
While at it, it would be great if you could add i18n here

Reply:
done